### DISCUSSION TITLE
我是小白想请教几个问题

### DISCUSSION THREAD

**Discussion by AQian423** [MaLuns/hexo-theme-async#26] [Q&A 疑难杂症] [answered]
![image](https://user-images.githubusercontent.com/106458443/196200939-aea699c6-084f-497f-98d4-43bb4c43c31a.png)
首页这个最近发布跑上面去了，怎么移下来呀，

还有我不知道怎么创建文章的分类

![image](https://user-images.githubusercontent.com/106458443/196201173-f77909d4-dfa7-418b-9535-7ec0a889706e.png)
底部运行多少了我的也不能正常显示

![image](https://user-images.githubusercontent.com/106458443/196201338-3b446816-4b7b-435c-98e1-66abd29d9bce.png)
links 添加友链列表信息后  运行网页不显示出来

![image](https://user-images.githubusercontent.com/106458443/196201520-a5416fc8-60a7-463c-bffb-af75cfbef6e9.png)
首页和归档页的浏览器标题不知道怎么修改

纯小白这么好看的主题能搭建起已经很开心了，哈哈。

**Comment by MaLuns** [comment, marked as answer]
最近发布在上面是因为没有填写 文章分类 [参考地址](https://hexo.io/zh-cn/docs/front-matter)
类似这样
``` yaml
--- 
categories: 记录类
---
```

标题一般是修改文章里的 title 
``` yaml
--- 
title: 你的标题
---
```
首页标题是  _config.yml 配置文件里

**Comment by MaLuns** [comment]
底部运行多少 ，我的配置这样的
``` yaml
footer:
  live_time:
    enable: true
    prefix: footer.tips
    start_time: 11/19/2019 17:00:00 #计时开始时间
```

友链的 那个应该是你配置错地方了，不是 banner 里的 links ,  是和 banner  平级 添加一个就可以了

**Reply by AQian423** [reply]
谢谢大佬，问题全部解决了，哈哈感谢感谢，辛苦你了
